blaze-client/src/main/scala/org/http4s/client/blaze/BlazeClientConfig.scala

```diff
@@ -11,8 +11,14 @@ import scala.concurrent.duration.Duration
 
 /** Config object for the blaze clients
   *
-  * @param idleTimeout duration that a connection can wait without traffic before timeout
-  * @param requestTimeout maximum duration for a request to complete before a timeout
+  * @param responseHeaderTimeout duration between the completion of a request
+  * and the completion of the response header.  Does not include time
```

Comment:
Response _prelude_ seems like better wording.

Reply:
I like prelude, but I think we're the only ones who have ever used that term.